docs/user/openstack/connect_nodes_to_ipv6_network.md

```diff
@@ -0,0 +1,59 @@
+# Connect workers nodes and Pods to IPv6 network
+
+To connect your workers at the time of installation you can use [additionalNetworkIDs](https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#additional-networks) parameter in the install config and set IPv6 network ID there:
+
+Example OpenShift install config:
+
+```yaml
+...
+compute:
+- name: worker
+  platform:
+    openstack:
+      additionalNetworkIDs:
+      - <ipv6_network_id>
+...
+```
+
+**Note**: To use Stateful IPv6 Networks, the arg `ip=dhcp,dhcpv6` needs to be included in the kernal args of the Worker nodes, otherwise the Nodes won't get an IPv6 address. Use the [procedure][add-kernel-args] to add kernel argument to the Nodes.
```

Comment:
The fact that DHCPv6 doesn't work out of the box feels like a bug honestly. We should probably reference a jira, in addition to mentioning the workaround with modifying the kernel arguments.

Reply:
Done. I created a bug in the installer